src/components/filtermenu/filtermenu.js

```diff
@@ -1,225 +1,222 @@
+    var value = elem.checked;
+    value = value ? value : null;
+    userSettings.setFilter(key, value);
+}
+function moveCheckboxFocus(elem, offset) {
+    var parent = dom.parentWithClass(elem, 'checkboxList-verticalwrap');
+    var elems = focusManager.getFocusableElements(parent);
+
+    var index = -1;
+    for (var i = 0, length = elems.length; i < length; i++) {
+        if (elems[i] === elem) {
+            index = i;
+            break;
         }
+    }
 
-        var html = '';
-
-        html += items.map(function (filter) {
-            var itemHtml = '';
-
-            var checkedHtml = isCheckedFn(filter) ? ' checked' : '';
-            itemHtml += '<label>';
-            itemHtml += '<input is="emby-checkbox" type="checkbox"' + checkedHtml + ' data-filter="' + filter.Id + '" class="' + cssClass + '"/>';
-            itemHtml += '<span>' + filter.Name + '</span>';
-            itemHtml += '</label>';
+    index += offset;
 
-            return itemHtml;
-        }).join('');
+    index = Math.min(elems.length - 1, index);
+    index = Math.max(0, index);
 
-        elem.querySelector('.filterOptions').innerHTML = html;
+    var newElem = elems[index];
+    if (newElem) {
+        focusManager.focus(newElem);
     }
-
-    function renderDynamicFilters(context, result, options) {
-        renderOptions(context, '.genreFilters', 'chkGenreFilter', result.Genres, function (i) {
-            // Switching from | to ,
-            var delimeter = (options.settings.GenreIds || '').indexOf('|') === -1 ? ',' : '|';
-            return (delimeter + (options.settings.GenreIds || '') + delimeter).indexOf(delimeter + i.Id + delimeter) !== -1;
-        });
+}
+function centerFocus(elem, horiz, on) {
+    import('scrollHelper').then(({ default: scrollHelper }) => {
+        var fn = on ? 'on' : 'off';
+        scrollHelper.centerFocus[fn](elem, horiz);
+    });
+}
+function onInputCommand(e) {
+    switch (e.detail.command) {
+        case 'left':
+            moveCheckboxFocus(e.target, -1);
+            e.preventDefault();
+            break;
+        case 'right':
+            moveCheckboxFocus(e.target, 1);
+            e.preventDefault();
+            break;
+        default:
+            break;
     }
-
-    function loadDynamicFilters(context, options) {
-        var apiClient = connectionManager.getApiClient(options.serverId);
-
-        var filterMenuOptions = Object.assign(options.filterMenuOptions, {
-
-            UserId: apiClient.getCurrentUserId(),
-            ParentId: options.parentId,
-            IncludeItemTypes: options.itemTypes.join(',')
-        });
-
-        apiClient.getFilters(filterMenuOptions).then(function (result) {
-            renderDynamicFilters(context, result, options);
-        });
+}
+function saveValues(context, settings, settingsKey) {
+    var elems = context.querySelectorAll('.simpleFilter');
+    var i;
+    var length;
+    for (i = 0, length = elems.length; i < length; i++) {
```

Comment:
```suggestion
    for (let i = 0, length = elems.length; i < length; i++) {
```

Reply:
great catch, I'll get that fixed in a minute

Reply:
Welp this is my mistake, `let` needs to be added to all the other for loops